src/internet_identity/src/main.rs

```diff
@@ -786,6 +789,44 @@ fn http_request(req: HttpRequest) -> HttpResponse {
     }
 }
 
+/// List of recommended security headers as per https://owasp.org/www-project-secure-headers/
+/// These headers enable browser security features (like limit access to platform apis and set
+/// iFrame policies, etc.).
+fn security_headers() -> Vec<HeaderField> {
+    vec![
```

Comment:
I also checked identity.raw.ic0.app on securityheaders.com and noticed we don't see _any_ security headers. See [here](https://securityheaders.com/?q=identity.raw.ic0.app&followRedirects=on). 

* Should we add also HSTS, X-Frame-Options, X-Content-Type-Options here?

Reply:
X-Frame-Options, X-Content-Type-Options are both there?
HSTS I missed because I did not check the raw page. Thanks!

Reply:
> X-Frame-Options, X-Content-Type-Options are both there?

Oh sure, sorry I got confused ;-) Thanks for adding HSTS!